server/libs/infra/src/search/schemas/asset.schema.ts

```diff
@@ -0,0 +1,37 @@
+import { CollectionCreateSchema } from 'typesense/lib/Typesense/Collections';
+
+export const assetSchemaVersion = 1;
+export const assetSchema: CollectionCreateSchema = {
```

Comment:
I assume the CLIP embedded vector will go into this schema when it is implemented, correct?

Reply:
Yes, they have a specific `vector` field type that we can use for it.